picus/r1cs/r1cs-grammar.rkt

```diff
@@ -286,12 +241,7 @@
     ; define internal function
```

Comment:
These comments feel useless to me. Let's instead describe the purpose of the internal function or just remove the comment entirely.